### PR TITLE
Adjust issuer URL in security-jwt guide

### PR DESCRIPTION
**Pull request by anhldbk**
I've updated links with 404 errors

### CODE REVIEW

File: docs/src/main/asciidoc/security-jwt.adoc [hunk 1]
```diff
@@ -303,10 +303,10 @@ Create a `security-jwt-quickstart/src/main/resources/application.properties` wit
 [source, properties]
 ----
 mp.jwt.verify.publickey.location=META-INF/resources/publicKey.pem #<1>
-mp.jwt.verify.issuer=https://quarkus.io/using-jwt-rbac #<2>
+mp.jwt.verify.issuer=https://quarkus.io/guides/security-jwt #<2>
 ----
 <1> We are setting public key location to point to a classpath publicKey.pem resource location. We will add this key in part B, <<Adding a Public Key>>.
-<2> We are setting the issuer to the URL string `https://quarkus.io/using-jwt-rbac`.
+<2> We are setting the issuer to the URL string `https://quarkus.io/guides/security-jwt`.
```
Comment (gsmet) [inline docs/src/main/asciidoc/security-jwt.adoc:309]: I'm not sure about this one tbh but my guess is that we should use some `https://example.com/url` as using a quarkus.io one is misleading.

Comment (sberyozkin) [reply docs/src/main/asciidoc/security-jwt.adoc:309]: @gsmet Right, maybe something like `https://example.com/issuer`.
@anhldbk FYI, the issuer claim URL value does not have to resolve to any specific site. 
I would not mind closing this PR as it is an important point, but if you'd like, then please create a PR for the quickstarts project first, update [these properties](https://github.com/quarkusio/quarkus-quickstarts/blob/master/security-jwt-quickstart/src/main/resources/application.properties), [this test](https://github.com/quarkusio/quarkus-quickstarts/blob/master/security-jwt-quickstart/src/test/java/org/acme/security/jwt/TokenSecuredResourceTest.java) and [this file](https://github.com/quarkusio/quarkus-quickstarts/blob/master/security-jwt-quickstart/src/test/java/org/acme/security/jwt/GenerateToken.java), note the PR there should go against the `development` branch. There should also be other references to the current issuer value in this doc too.

Comment (anhldbk) [reply docs/src/main/asciidoc/security-jwt.adoc:309]: @gsmet @sberyozkin How about change the link to `https://quarkus.io` ?

Comment (sberyozkin) [reply docs/src/main/asciidoc/security-jwt.adoc:309]: @anhldbk IMHO if we are to change it then we need to break the association with `quarkus.io` completely, as I've said, this value has nothing to do with the `quarkus.io`

Comment (anhldbk) [reply docs/src/main/asciidoc/security-jwt.adoc]: @sberyozkin ok. I fixed. PTAL

**Review by sberyozkin — APPROVED** [review]
@anhldbk thanks, I've also opened https://github.com/quarkusio/quarkus-quickstarts/pull/670

**Comment by gsmet** [comment]
@sberyozkin please get the commits squashed before merging next time. There's no point having 3 commits for this change. Thanks.

**Pull request merged**